update Wallet Recovery to 2.0 (#1182)
zkSNACKs/WasabiDoc: docs/using-wasabi/WalletRecovery.md

Wasabi uses the [BIP 39](https://github.com/bitcoin/bips/blob/master/bip-0039.mediawiki) mnemonic code for generating [BIP 32](https://github.com/bitcoin/bips/blob/master/bip-0032.mediawiki) hierarchical deterministic wallets.
**Both** your password **and** the generated 12 mnemonic recovery words are the necessary secrets to recover your wallet.
The recovery words are spell checked based on the [official wordlist](https://github.com/bitcoin/bips/blob/master/bip-0039/english.txt), with the right word shown below the text box.

> 1. To recover a wallet, go to `Add Wallet`.

Reviewer: ```suggestion
To recover a wallet:

1.Click on `Add Wallet` at the bottom left corner of the main view.
```

**Both** your password **and** the generated 12 mnemonic recovery words are the necessary secrets to recover your wallet.
The recovery words are spell checked based on the [official wordlist](https://github.com/bitcoin/bips/blob/master/bip-0039/english.txt), with the right word shown below the text box.

1. To recover a wallet, go to `Add Wallet`.

![Wasabi Wallet Add Wallet](/AddWallet.png "Wasabi Wallet Add Wallet")

2. Click on `Recover a wallet`.

3. Give the wallet a name for future recognition of the wallet.

![Wasabi Wallet Name Wallet](/AddWalletWalletName.png "Wasabi Wallet Name Wallet")

4. Type in the Recovery Words in the correct order and click `Continue`.

![Wasabi Wallet Recover Wallet](/RecoverWallet.png "Wasabi Wallet Recover Wallet")

5. Type in the Password.

![Add Wallet Add Password](/AddWalletAddPassword.png "Add Wallet Add Password")

Now Wasabi will recover your wallet, and [load the wallet](/using-wasabi/WalletLoad.md).
After a short loading period, you can use Wasabi as usual.
With this, only the private keys are imported, but not the labels of your addresses.

> 

Reviewer: ```suggestion
```

:::danger
At recovery, Wasabi is unable to check if your password is correct or not.
If you type a wrong password a completely different wallet will be recovered.
:::

> You can also use the `Advanced Recovery Options` to specify an account key path of the HD wallet structure and the gap limit.

Reviewer: ```suggestion
You can also use the `Advanced Recovery Options` to specify the account key path of the HD wallet structure and the gap limit.
```

## Back up Wallet File and Password

Wasabi Wallet creates a backup of your wallet file inside `WalletBackups` in your [Wasabi data folder](/FAQ/FAQ-UseWasabi.md#where-can-i-find-the-wasabi-data-folder).
If you have done a backup of this file, then you can import it at `Add Wallet`, by clicking on `Import a wallet`.

> Alternitavely: copy the wallet file to the `Wallets` folder, and upon the next restart of Wasabi, it will show this wallet in the `NavBar`, from there you can open it as usual.

Reviewer: ```suggestion
Alternitavely: copy the wallet file to the `Wallets` folder, and upon the next restart of Wasabi, it will show this wallet in the `NavBar`, from there you can open it as usual.

```
